### ISSUE TITLE
Eccentric Tome and EnderIO book

### ISSUE THREAD

**Issue by kiaen1** [DarkPacks/SkyFactory-5#9]
## Context

The EnderIO book in the eccentric tome shouldn't be there, (The book is Literally empty, it looks like they started it either as a test or forgot they had started it.

![image](https://github.com/user-attachments/assets/d57da80e-4b11-4368-a62d-b5986bba0b8c)

**Comment by Darkosto** [comment]
Yeah, I'm going to leave it here for now as it would be redoing the recipes and if they add to it then it will already be setup. Thanks for the report!
